Web/CaseStudyAssignment-4/CaseStudyAssignment-4/WebContent/addCar.jsp

```diff
@@ -0,0 +1,80 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+    
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+		<title>Car Portal</title>
+		<link rel="stylesheet" type="text/css" href="css/style.css">
+		<link rel="stylesheet" type="text/css" href="css/styleLogin.css">
+	</head>
+	<body>
+		<header> 
+		<br />
+		<span id="logo">Car Dekho</span> 
+		<%
+	 		if (session.getAttribute("username") != null) {
+		%>
+		<div id="user">
+			<%=session.getAttribute("username")%>&nbsp;&nbsp;<a href="login.jsp"> Logout!!</a>
+		</div>
+		<%
+			}
+		%> 
+		</header>
+		<nav>
+		<%
+			if (session.getAttribute("username") != null) {
```

Comment:
ACTION: No need to check again username to render home link, As above you already checking session to render logout. in same block you can write code for home link.